File: easy/pro-ballpark-orders/BallparkOrders.java
import java.util.Map;
import java.util.Scanner;
import java.math.BigDecimal;

public final class BallparkOrders {
    public static void main(String[] args) {
        final var menu = Map.of(
            "Nachos", 6, "Pizza", 6, "Cheeseburger", 10, "Water", 4, "Coke", 5);
        
        final var tax = new BigDecimal("1.07");
        
        final var sum = new Scanner(System.in)
            .tokens()
            .mapToInt(item -> menu.getOrDefault(item, menu.get("Coke")))
            .sum();
            
        final var totalCost = new BigDecimal(sum).multiply(tax);
        
        System.out.print(totalCost.setScale(2));
    }
}

File: easy/pro-ballpark-orders/ballpark_orders.c
#include <stdio.h>
#include <string.h>
#define MAX 300

int main() {
  char order[MAX];
  fgets(order, MAX, stdin);
  
  double totalCost = 0;
  char menu[5][14] = {"Nachos", "Pizza", "Cheeseburger", "Water", "Coke"};
  int price[5] = {6, 6, 10, 4, 5};
  
  for (char *item = strtok(order, " "); item; item = strtok(NULL, " ")) {
  	 int itemCost = 5; 
  	 for (int i = 0; i < 5; ++i) {
  	 	if (strcmp(item, menu[i]) == 0) {
  	 		itemCost = price[i];
  	 		break;
  	 	}
  	 }
  	 totalCost += itemCost;
  }
  
  totalCost *= 1.07;
  printf("%.2f", totalCost);
  return 0;
}
